cdn-clone.js

```diff
@@ -0,0 +1,29 @@
+function loadTXT(url, callback) {
+
+    var xobj = new XMLHttpRequest();
+    xobj.open('GET', url, true);
+    xobj.onreadystatechange = function () {
+        if (xobj.readyState == 4 && xobj.status == "200") {
+            callback(xobj.responseText);
+        }
+    };
+    xobj.send(null);
+}
+
+function updateStatus(last_updated) {
+  var now_date = new Date();
+  var last_updated_date = new Date(last_updated.trim())
+  var THIRTY_MIN = 30 * 60 * 1000; /* ms */
+  var cloning_status = document.getElementsByClassName("cloning")[0].getElementsByClassName("status")[0]
```

Comment:
`status` feels like a potentially too generic classname which might collide with other things.  Can we rename to `cdn-status`